website/src/docs/hotchocolate/performance/persisted-queries.md

```diff
@@ -2,12 +2,237 @@
 title: "Persisted queries"
 ---
 
-Hi,
+This guide will walk you through how standard persisted queries works and how you can set them up with the Hot Chocolate GraphQL server.
 
-We're currently working on the version 11 documentation. Probably right now at this very moment. However, this is an open-source project, and we need any help we can get! You can jump in at any time and help us improve the documentation for hundreds or even thousands of developers!
+# What they are
 
-In case you might need help, check out our slack channel and get immediate help from the core contributors or the community itself.
+Allows you to pre-register all required queries/mutations from your client on your GraphQl server.
 
-Sorry for any inconvenience, and thank you for being patient!
+## Why to use
 
-The ChilliCream Team
+There are two main reasons why to use persisted queries:
+
+- Performance - Payload contains only Hash and variables. This will reduce the size of your client application since queries can be removed from the client code at the build time.
+
+- Security - By using persisted queries, the server will accept only known queries / mutations and refuse all others that are not part of persisted "Allowed-List". Useful mainly for public APIs. 
+
+## How it works
```

Comment:
I would maybe move this up to the introduction. In general, I would envision a ToC like the following:
```
# Introduction
what they are and how they work
## Benefits
self-explanatory
## Limitations
self-explanatory
# Usage
things that don't change between storages, i.e. UsePersistedQueryPipeline
## Filestystem
self-explanatory
## InMemory
self-explanatory
## Redis
self-explanatory
## Hashing Algorithms
self-explanatory
# Client-side implementation
links to relay, apollo, etc.
```

Reply:
Made some adjustments and move tops